### ISSUE TITLE
Useless elytra

### ISSUE THREAD

**Issue by Whataheckisasonic** [Ladysnake/Requiem#585]
none of the existing mobs can use the elytra. its literally impossible to use. all mobs or atleast some of them should be able to use the elytra.

**Comment by Touch-Night** [comment]
But vanilla minecraft mobs can't use elytra either!

**Comment by ShadeTDW** [comment]
But that's probably because they aren't smart enough though.

**Comment by kristibektashi** [comment]
> But vanilla minecraft mobs can't use elytra either!

Yes they can with commands, so they should be able to with this mod as well

**Comment by AFinalWord** [comment]
> > But vanilla minecraft mobs can't use elytra either!
> 
> Yes they can with commands, so they should be able to with this mod as well

but thats with commands, which isnt really how you play the game normally since its technically cheating
if an entity gets an elytra normally (for example, if a zombie picks one up) they cant fly with it, normal entities dont know how to use elytras, so why would possessed mobs know?

**Comment by kristibektashi** [comment]
> > > But vanilla minecraft mobs can't use elytra either!
> > 
> > 
> > Yes they can with commands, so they should be able to with this mod as well
> 
> but thats with commands, which isnt really how you play the game normally since its technically cheating if an entity gets an elytra normally (for example, if a zombie picks one up) they cant fly with it, normal entities dont know how to use elytras, so why would possessed mobs know?

You said it yourself. They are possessed. Possessed by a player that actually knows how to use an elytra unlike the mobs themselves which simply aren't smart enough to do so on their own.

**Comment by Eman-3600** [comment]
> You said it yourself. They are possessed. Possessed by a player that actually knows how to use an elytra unlike the mobs themselves which simply aren't smart enough to do so on their own.

Players know how to use an elytra in their normal form, yes, but even humanoid mobs have slightly different proportions and don't move the same (zombies' leg muscles are slower for instance). So it could be that mobs don't have the right build to use them.

For a more technical reason, no mob has a fall-flying model, and it would be a big investment to provide ones for them. Just leaving them be without a fall-flying model yet allowing elytra use would seem rather unpolished.

Would it be a cool feature? Yeah. Is it feasible? Unsure. Would it make sense either way? Yes.

**Comment by kristibektashi** [comment]
> > You said it yourself. They are possessed. Possessed by a player that actually knows how to use an elytra unlike the mobs themselves which simply aren't smart enough to do so on their own.
> 
> Players know how to use an elytra in their normal form, yes, but even humanoid mobs have slightly different proportions and don't move the same (zombies' leg muscles are slower for instance). So it could be that mobs don't have the right build to use them.
> 
> For a more technical reason, no mob has a fall-flying model, and it would be a big investment to provide ones for them. Just leaving them be without a fall-flying model yet allowing elytra use would seem rather unpolished.
> 
> Would it be a cool feature? Yeah. Is it feasible? Unsure. Would it make sense either way? Yes.

I do think there should be a config option that is disabled by default so that players possessing mobs can use elytras even without the proper models.
